src/components/WhatIsNew/WhatIsNew.spec.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import WhatIsNew from './WhatIsNew';
+
+describe('<WhatIsNew />', () => {
+  let wrapper;
+
+  beforeEach(() => {
```

Comment:
Also you can create a test that will check that function `test` was called using jest.spy